concent_api/core/models.py

```diff
@@ -637,14 +637,14 @@ def clean(self) -> None:
                 'provider_eth_account': 'Provider ethereum account address must be diffrent than task owner key'
             })
 
-        if not isinstance(self.amount_pending, int) or self.amount_pending < 0:
+        if self.amount_pending < 0:
```

Comment:
I know that this is not related with that changes but as we are here, what about condition when it is equal 0? Neither the condition nor error message cover that case (so on of those should be fixed).
Apart from that, I think it should work still if you check that value type is either `int` or `Decimal`.

Reply:
I think that `amount_pending` can be `0` - if all claims have been satisfied.

Reply:
Each of those validations is not actually needed to be performed there, if you want to validate if value is grater than 0 you can set `MinValueValidator(0)` directly on the model. I thought we are doing this for custom error messages - if not, what is the reason for keeping this custom validation?

Reply:
@cameel said the same thing on Friday, I can change it to django Validator then.

Reply:
Ok, please do that then.

Reply:
Done